drivers/scheduler/k8s/k8s.go

```diff
@@ -669,7 +698,63 @@ func (k *K8s) createVolumeSnapshotRestore(specObj interface{},
 	return nil, nil
 }
 
-func (k *K8s) createCoreObject(spec interface{}, ns *v1.Namespace, app *spec.AppSpec) (interface{}, error) {
+func (k *K8s) addSecurityAnnotation(spec interface{}, configMap *v1.ConfigMap) {
+	logrus.Infof("Config Map details:\n %v:", configMap.Data)
+	if obj, ok := spec.(*v1.PersistentVolumeClaim); ok {
+		if obj.Annotations == nil {
+			obj.Annotations = map[string]string{}
+		}
+		obj.Annotations[secretName] = configMap.Data["secret_name"]
+		obj.Annotations[secretNamespace] = configMap.Data["secret_namespace"]
```

Comment:
use `const` for "secret_name" and "secret_namespace".

Reply:
Fixed.